homework/style.css

```diff
@@ -1,3 +1,144 @@
+:root {
+  --first: rgb(255, 255, 255);
+  --second: coral;
+  --third: black;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  margin-top: 12px;
+}
+
+.container {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+  align-items: center;
```

Comment:
On a mobile device, `justify-content` works better than `align-items` here. I always use this site to help me with flexbox:

https://css-tricks.com/snippets/css/a-guide-to-flexbox/